index.html

```diff
@@ -16,9 +16,14 @@
 <body>
     <section class="main-page">
         <header class="header">
-            <a href="" class="logo"><h3>Ali Ahmad Karimi</h3></a>
+            <a href="#" class="logo"><h3>Ali Ahmad Karimi</h3></a>
             <nav class="toolbar">
-                <img src="images/Icon-Menu.png" alt="Menu Icon">
+                <img src="images/Icon-Menu.png" alt="Menu Icon" class="img-3">
+                <ul>
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                </ul>
             </nav>
```

Comment:
- [ ] `optional`
- [ ] Kindly fix the navbar as similar as possible to the figma template. please see screenshots  below

`figma template`
![Screenshot from 2022-07-07 10-16-47](https://user-images.githubusercontent.com/66230306/177874168-fc344598-b013-4c38-a828-0201377ca294.png)

`your template`
![Screenshot from 2022-07-07 10-28-56](https://user-images.githubusercontent.com/66230306/177874473-e58e342c-e052-45fa-996e-53ea6403db63.png)